windows/src/developer/kmcmpdll/Compiler.cpp

```diff
@@ -559,7 +560,7 @@ BOOL CompileKeyboardHandle(HANDLE hInfile, PFILE_KEYBOARD fk)
 
 	delete str;
 
-	return TRUE;
+  return CheckKeyboardFinalVersion(fk);
```

Comment:
This line sets the file's version at the very end of compilation?

Reply:
This just prints an info report on the minimum version requirement that was determined by the compiler, if `KF_AUTOMATICVERSION` is specified. See versioning.cpp for implementation.